apps/style/javaIde/style.scss

```diff
@@ -0,0 +1,16 @@
+.container {
+  width: 95% !important;
```

Comment:
Willing to be convinced otherwise: 
I've run into problems using `!important` in the past where we end up with `!important` chaining and lots of confusing bugs. I'd recommend staying away from it if it can be avoided.

Reply:
do you know a way to ensure the component takes up the entire screen that doesn't use `!important`? Our default is only part of the screen and I want it to take up the entire width.

Reply:
Hmm do we have the width set with `!important` somewhere else? If so, this will be the only way to override it. If not, it may just be an issue of ordering. This style file should be imported last, but it might not be. You could try applying the styling directly on the component - that will be the last style applied and will override anything that isn't using `!important`

Reply:
huh I tried without `!important` and it worked (I had assumed it wouldn't because I'd needed !important in a previous page that needed full width). I don't think I can add it on the component since I need the style applied on `.container`. So maybe we don't have it set elsewhere with !important